custom/fri/models.py

```diff
@@ -1,60 +0,0 @@
-from __future__ import absolute_import
-from __future__ import unicode_literals
-from dimagi.ext.couchdbkit import *
-
-PROFILE_A = "A"
-PROFILE_B = "B"
-PROFILE_C = "C"
-PROFILE_D = "D"
-PROFILE_E = "E"
-PROFILE_F = "F"
-PROFILE_G = "G"
-PROFILE_H = "H"
-PROFILES = [PROFILE_A, PROFILE_B, PROFILE_C, PROFILE_D, PROFILE_E, PROFILE_F, PROFILE_G, PROFILE_H]
-PROFILE_DESC = {
-    PROFILE_A: "A - HIV+",
-    PROFILE_B: "B - ART non",
-    PROFILE_C: "C - IDU",
-    PROFILE_D: "D - PSE/CSV",
-    PROFILE_E: "E - Top",
-    PROFILE_F: "F - Bottom",
-    PROFILE_G: "G - General",
-    PROFILE_H: "H - Other",
-}
-
-
-class FRIMessageBankMessage(Document):
```

Comment:
Should there be a management command to delete these old docs when we delete these?

Reply:
Hmm, yeah, probably - although then we wouldn't ever be able to revert.  I'll ask Jeremy what he thinks about that.  Might have to provide a dump or something.

Reply:
i started an email thread about this. depending on the answer to that, all the fri_* fields on the SMSBase class might also go